PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalLineItem.kt

```diff
@@ -0,0 +1,76 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Line item for PayPal checkout flows.
+ *
+ * @property kind Indicates whether the line item is a debit (sale) or credit (refund) to the
+ * customer.
+ * @property name Item name to display. Maximum 127 characters.
+ * @property quantity Number of units of the item purchased. This value must be a whole number and
+ * can't be negative or zero.
+ * @property unitAmount Per-unit price of the item. Can include up to 2 decimal places. This value
+ * can't be negative or zero.
+ * @property description Item description to display. Maximum 127 characters.
+ * @property imageUrl The image URL to product information.
+ * @property productCode Product or UPC code for the item. Maximum 127 characters.
+ * @property unitTaxAmount Per-unit tax price of the item. Can include up to 2 decimal places.
+ * This value can't be negative or zero.
+ * @property upcCode UPC code of the item.
+ * @property upcType UPC type of the item.
+ * @property url The URL to product information.
+ */
+
+@Parcelize
+data class PayPalLineItem
+@JvmOverloads constructor(
+    val kind: PayPalLineItemKind,
+    val name: String,
+    val quantity: String,
+    val unitAmount: String?,
```

Comment:
```suggestion
    val unitAmount: String,
```

I think this value is required

Reply:
Whoops, I removed the default value but kept the type as nullable. Just updated this.